Workiva/wf-uicomponents
# Prevent rendering when mouse wheel does not change position

Previously, when you scrolled to unloaded content at the boundaries (top or bottom of page) and kept scrolling (even though there was nowhere to scroll to), the content would not load. Now, if you scroll at the boundaries, ignore it and load the content like you would expect.
## How to QA

``` bash
git fetch && git checkout loading-time-with-mousewheels-at-boundary && ./init.sh && grunt qa
```

[ ] Verify all tests pass.
[ ] Open ScrollList example in webpage that pops up due to `grunt qa` and scroll down fast until you hit unloaded page (white page and then flicker of content). Scroll continuously up until you hit the first page and keep scrolling up. Verify that page one loads (rather than staying at the white, "rendered" phase, as it did previously).
